apps/rush-lib/src/data/RushConfiguration.ts

```diff
@@ -186,10 +199,25 @@ export default class RushConfiguration {
       }
       currentFolder = parentFolder;
     }
+
     throw new Error('Unable to find rush.json configuration file');
```

Comment:
>throw new Error('Unable to find rush.json configuration file'); [](start = 4, length = 63)

If Rush is invoked outside of a repo folder, then there will not be any "rush.json" in any parent folder.  This is an expected situation.  If the code needs to test that, it should NOT have to do that by catching an unexceptional exception; instead, we should change the API to e.g. tryFindRushJsonLocation() which returns undefined if the path cannot be found.